plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/totals/shipping/shipping-rate-selector.tsx

```diff
@@ -15,17 +19,32 @@ export interface ShippingRateSelectorProps {
 	shippingRates: CartResponseShippingRate[];
 	isLoadingRates: boolean;
 	isAddressComplete: boolean;
+	shippingAddress: CartResponseShippingAddress;
 }
 
 export const ShippingRateSelector = ( {
 	hasRates,
 	shippingRates,
 	isLoadingRates,
 	isAddressComplete,
+	shippingAddress,
 }: ShippingRateSelectorProps ): JSX.Element => {
 	const legend = hasRates
 		? __( 'Shipping options', 'woocommerce' )
 		: __( 'Choose a shipping option', 'woocommerce' );
+
+	const formattedLocation = formatShippingAddress( shippingAddress );
+	const noResultsMessage = (
+		<>
+			{ __( 'No delivery options available for ', 'woocommerce' ) }
+			<strong>{ formattedLocation }</strong>
+			{ __(
+				'. Please verify the address is correct or try a different address.',
+				'woocommerce'
+			) }
+		</>
+	);
+
```

Comment:
Use interpolate element here instead of stitching strings like this.

Reply:
@senadir Thank you for the suggestion! I have updated the code.